apps/www/components/Front/index.js

```diff
@@ -46,13 +49,27 @@ const styles = {
   }),
 }
 
-export const RenderFront = ({ isEditor, front, nodes }) => {
+export const RenderFront = ({ front, nodes }) => {
   const { t } = useTranslation()
+  const { isEditor, hasAccess } = useMe()
+  const { addAudioQueueItem, isAudioQueueAvailable } = useAudioQueue()
+  const { toggleAudioPlayer } = useAudioContext()
+
+  const showPlayButton = hasAccess && isAudioQueueAvailable
```

Comment:
shouldn't this be `isReadaloud` instead of `isAudioQueueAvaiblabe` ?

Reply:
for the front we use a query to get audio data that's only available if the queue is available … we might refactor this to put play buttons on the marketing front, but after the release.